PITCH.md

```diff
@@ -0,0 +1,60 @@
+# Value Matrix
+
+This section will explain how Standard intends to create value for different stakeholders.
+It is essential to have an idea of who they are, so let's introduce:
+
+**_The Software Sponsor_**
+Makes resources available in return for the expectation of future benefits.
+
+**_The Provider of Automation_**
+Sets up and maintains the automation alongside the SDLC.
+A helpful analogy would be the person who sets up and maintains the conveyor belt, which moves features to production.
+
+**_The Consumer of Automation_**
+Consumes and co-maintaines the automation alongside the SDLC.
+A helpful analogy would be that this person not only uses and configures our conveyor belt, but is also capable of maintaining it.
```

Comment:
The previous analogy says that the person also maintains the conveyour belt, so in essence the two mean the same thing. I'm pretty sure that was not your intention.